org/wildfly/undertow/http-invoker/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.undertow.http-invoker
+
+DESCRIPTION: Entry point for remote http invoker, supporting ejb over http
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: false
+
+PRIVATE: false
```

Comment:
I don't see any outside use of this in https://github.com/wildfly/wildfly/pull/9858/files. Should it be PRIVATE: true?

I hope so as exposing PathHandler as the API doesn't seem great.